pivottablejs.py

```diff
@@ -8,22 +8,52 @@
 template = """
 <!DOCTYPE html>
 <html>
-    <head>
+    <head id="head-pivottable">
         <title>PivotTable.js</title>
 
         <!-- external libs from cdnjs -->
-        <link rel="stylesheet" type="text/css" href="%(static)s/pivot-ajax/c3.min.css">
-        <script type="text/javascript" src="%(static)s/pivot-ajax/jquery.min.js"></script>
-        <script type="text/javascript" src="%(static)s/pivot-ajax/jquery-ui.min.js"></script>
-        <script type="text/javascript" src="%(static)s/pivot-ajax/d3.min.js"></script>
-        <script type="text/javascript" src="%(static)s/pivot-ajax/jquery.csv-0.71.min.js"></script>
-        <script type="text/javascript" src="%(static)s/pivot-ajax/c3.min.js"></script>
+        <script type="text/javascript">
+        var static_url = window.location.href.split('/');
+        static_url = static_url.slice(0,6);  // keep up to app 'notebook'
```

Comment:
Did you try this extension with a local jupyter instance? This package should handle the AEN case but also the plain jupyter notebook and I am pretty sure this will break with a naive jupyter.